riotfile.py

```diff
@@ -626,10 +627,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         "DATADOG_SERVICE_NAME": "test.flask.service",
                         "DATADOG_PATCH_MODULES": "jinja2:false",
                     },
-                    pkgs={
-                        "flask": ["~=0.12.0"],
-                        "pytest": "~=3.0",
-                    },
+                    pkgs={"flask": ["~=0.12.0"], "pytest": "~=3.0", "more_itertools": "<8.11.0"},
```

Comment:
WHy are we not able to define `mode_itertools` dependency just once at the main Venv for flask?

Reply:
imo, probably not a good idea since it's just Python 3.5 that is impacted

Reply:
okay